cmd/fry.go

```diff
@@ -108,7 +108,7 @@ func doParseSha1List(config *types.Config) (exitCode int, err error) {
 	}
 
 	log.WithField("path", config.Path).Info("Beginning parsing of file into sha1 type")
-	sha1s, err := parse.ParseSha1File(config.Path)
+	sha1s, err := parse.Sha1File(config.Path)
```

Comment:
Suggestion from go, basically `parse.ParseSha1File` is a stutter, this is `parse.Sha1File` which reads perfectly.

Reply:
Makes sense.